### PR TITLE
parallel processing foundry output files

### PR DESCRIPTION
**Pull request by TilakMaddy**
building the ast can be a heavy process so we should take advantage of multiple cores if available in the machine.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aderyn"
-version = "0.0.7"
```
Comment (alexroan) [inline Cargo.toml:3]: Can you bump back down to 0.0.7? PRs are merged into `dev`, and we cut the release from there. It just so happens that I've bumped this one early, so the next release is 0.0.7.

File: README.md [hunk 2]
```diff
@@ -164,3 +164,7 @@ This project exists thanks to all the people who [contribute](/contributing.md).
 [license-shield]: https://img.shields.io/github/license/cyfrin/aderyn?logoColor=%23fff&color=blue
 [license-url]: https://github.com/cyfrin/aderyn/blob/master/LICENSE.txt
 [linkedin-shield]: https://img.shields.io/badge/-LinkedIn-black.svg?style=for-the-badge&logo=linkedin&colorB=555
+
```
Comment (alexroan) [inline README.md:167]: We can remove this too.

**Review by alexroan — CHANGES_REQUESTED** [review]
Thank you so much for these PRs!!!

Just a couple of changes to request on this one.

**Comment by alexroan** [comment]
What's the best way to reach out to you @TilakMaddy ? I've pinged you on discord. If you've got Twitter, you can reach me at @alexroan

File: Cargo.toml (hunk 1 above)
Comment (TilakMaddy) [reply Cargo.toml]: Got it ! I bumped it back down

**Comment by TilakMaddy** [comment]
@alexroan you are too kind! I'm flattered :)  Honestly, @PatrickAlphaC & Team at Cyfrin has done so much ! - I feel like I owe you

**Review by alexroan — APPROVED** [review]

**Pull request merged**